libnmstate/schemas/operational-state.yaml

```diff
@@ -456,3 +483,24 @@ definitions:
         type: array
         items:
           type: string
+  vlan-filter:
```

Comment:
In the reply to the last patch set, you indicate your desire to also model the uplink port - despite that not being needed for kubevirt. I'm OK with that.

The implementation for that would re-use the vlan filtering definition.

This can be called multiple things, but I'd really rather call it what it is. A vlan filter. Please suggest an alternate name if you dislike this one so much.

Yes, it could be re-used by OVS , the uplink linux bridge port, etc. Since we've agreed a model, I think re-using it whenever possible is preferable.

Reply:
In my view, vlan-filter is the implementation and the model is port vlan support.
`bridge-port-vlan` sounds more appropriate, but you can suggest anything that describes what this represents.

Reply:
Fair enough.

Reply:
Did I missed something?